Documentation/ExtensionArchitecture/ComposerJson/Index.rst

```diff
@@ -274,24 +270,10 @@ Example for extension key **bootstrap_package**:
       }
    }
 
-
-As all extensions available in the TER can be installed
-with `composer require typo3-ter/ext-key`, this makes sure that
-there will be no conflicts with packages installed or required
-via Packagist or from another source.
-
-Since the TER Composer repository is deprecated and not all extensions
-must be available in TER, this property is usually not required.
-
-Properties no longer used
-=========================
-
-version
--------
-
-Was used in earlier TYPO3 versions.
-For TYPO3 versions 7.6 and above you should not use the version property.
-The version for the extension is set in the file :ref:`ext_emconf.php <ext_emconf-php>`.
+This was used previously as long as the TER Composer repository was 
```

Comment:
I would remove the "and not all extensions need to be available in the TER" part, as this makes the documentation confusing: it doesn't necessarily have anything to do with the reason for removing the replace attribute, and might give the extension author the idea of not publishing the extension in the TER, which is undesirable. Instead, I would stick with your original comment on this PR and just explain what the attribute was used for and why it is no longer necessary.

Terminology: TER Composer **r**epository -> TER Composer **R**epository (see on https://get.typo3.org/misc/composer/repository)

Reply:
> the idea of not publishing the extension in the TER, which is undesirable

Actually that is very desirable IMO if it means that the focus is on Packagist, thus the regular Composer ecosystem. If one still wants to support Legacy Mode, he is free to continue publishing to TER however.

Reply:
Why should it be preferred to host it only on Packagist instead of Packagist and TER? Being available for _all_ modes should be desirable in my eyes. In addition, the TER page offers additionally information regarding the extension besides purely hosting it.

Reply:
Because the Legacy Mode is bound to disappear at some point. Making packages available in Composer Mode only can help fasten the process. :-)

Reply:
If you are not publishing your extension at https://extensions.typo3.org/ but only on Packagist, is it still discoverable at https://extensions.typo3.org/?

Reply:
Sorry comment got hiccuped in resolving: This discussion proves that this sentence might create a lot of confusion. However this sentence itself covers information for an edge case only which should probably be known for everybody running in the edge case (e.g. knowing what a composer package is). So from a TYPO3 perspective it won't hurt to remove it.

That said, I fully agree with all of mbrodala arguments, I just don't see any worth in upholding this sentence.

n.b.: I just moved that stuff as an atomic change, I didn't want to refactor everything :-P

Reply:
fyi, we are planing a feature for composer supported extensions only see https://git.typo3.org/services/t3o-sites/extensions.typo3.org/ter/-/issues/544